api_tests/src/shared.ts

```diff
@@ -623,6 +678,30 @@ export async function saveUserSettingsBio(api: API): Promise<LoginResponse> {
   return saveUserSettings(api, form);
 }
 
+// Feedback request: is this an ideal way to do it? Send back the whole form, even if changing just a couple values?
+//  In other words, is there some built-in client way to cline a read of Localuser into a write of SaveUserSettings object?
+//  ToDo: it seems if you revise nothing on a profile, this throws "user_already_exists"?
```

Comment:
Its not necessary to send the current entire local user, only the ones that have changed. 2ndly, this function is also unused.

Reply:
All the unused are related to me really wanting to get this into main on Friday to work on more over the weekend...